lib/src/main/java/adminlte/entity_list_table/communication/http/tables/AbstractTable.java

```diff
@@ -46,4 +44,55 @@ protected void addColumn(ColumnDefinitionInterface column) {
     public String getPaginationParameter() { return this.paginationParameter; }
 
     public String getSearchParameter() { return this.searchParameter; }
+
+    public List<ColumnDefinitionInterface> getCsvColumns() { return new ArrayList<ColumnDefinitionInterface>(); }
```

Comment:
Honestly, I don't think that it makes a good sense to have a separate method for getCsvColumns. Why don't we use columns defined in the constructor/defineColumns method?

Reply:
We might not want to have the same info in csv as we do in the table on the page. Flexibility doesn't hurt. And if one needs, he can easily just make the return of this function the defined columns ones.

Reply:
Made current table's columns as default